<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1536837497109' stop='1536837534933'>
    <name>Test the Login Functionality with all posible scenario</name>
    <title>Test the Login Functionality with all posible scenario</title>
    <test-cases>
        <test-case start='1536837497109' status='failed' stop='1536837534925'>
            <name>Login with valid Username and Password</name>
            <title>Login with valid Username and Password</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (C:\Users\pawan.singh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at ontimeout (timers.js:424:11)
                    at tryOnTimeout (timers.js:288:5)
                    at listOnTimeout (timers.js:251:5)
                    at Timer.processTimers (timers.js:211:10)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>